local_vars/local_vars.cc

```diff
@@ -70,7 +71,22 @@ class LocalNameInserter {
             }
             unique_ptr<ast::Reference> refExpImpl(refExp);
             arg.release();
-            namedArgs.emplace_back(nameArg(move(refExpImpl)));
+            auto named = nameArg(move(refExpImpl));
+            for (auto &prev : namedArgs) {
```

Comment:
This is accidentally quadratic; not sure if we care to use a hashmap to avoid that.

Reply:
FWIW, this is the same logic we used indirectly before: it was just hidden behind `enterMethodArgumentSymbol`, which was called for every argument and would traverse all of the existing arguments first, so I don't believe this should be any slower than the current state.

I'm a bit nervous that for the common case of <4 or so arguments, the overhead of allocating a hashmap here would be a greater hit to performance than walking a short array, but I also don't want to make a specific decision one way or another without some data—maybe I can make an issue to investigate further once I've depleted fuzz bugs.